search-service/src/main/kotlin/com/egm/stellio/search/entity/web/EntityOperationHandler.kt

```diff
@@ -98,7 +103,12 @@ class EntityOperationHandler(
     suspend fun upsert(
         @RequestHeader httpHeaders: HttpHeaders,
         @RequestBody requestBody: Mono<String>,
-        @RequestParam(required = false) options: String?
+        @RequestParam(required = false) options: String?,
```

Comment:
a bit strange to have the `options` query parameter separated. for consistency, it should now be retrieve from `queryParams` (to be also checked in some other places)